### ISSUE TITLE
[Bug]: Disney+ crashes

### ISSUE THREAD

**Issue by LorenzoMorelli** [PlayCover/PlayCover#692]
### Describe the bug

When I try to open the app Disney+, it opens for an instant and then crash immediately.

### Steps to reproduce

Install Disney+.ipa with playcover and then try to open it.

### Expected behaviour

I would expect to use the app without crashing.

### Crash log

```shell
Unfortunately I was not able to get the crash log. If anyone can tell me how to I will instantly.
```

### What version of PlayCover are you using?

2.0.2

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by Amachik** [comment]
it is not playcover problem. first, u need to have disabled sip, and second, u need to use older version of disney if i remember correctly

**Comment by LorenzoMorelli** [comment]
Thank you for your advice,
I tried to disable sip but the crash is still there.
Can you tell me which version (or even better to share) of the app I should use?

Even if you said it is not a playcover problem, since I finally got the crash log I share it here below
```
/usr/bin/lldb -o run /Users/moremac/Library/Containers/io.playcover.PlayCover/Disney+.app/Dominguez -o exit
(lldb) target create "/Users/moremac/Library/Containers/io.playcover.PlayCover/Disney+.app/Dominguez"
Current executable set to '/Users/moremac/Library/Containers/io.playcover.PlayCover/Disney+.app/Dominguez' (arm64).
(lldb) run
2023-01-10 18:53:37.381871+0100 Dominguez[1469:10438] GoogleCast framework version 4.6.0.16.10
[SwordRPC] Discord not detected
2023-01-10 18:53:37.547939+0100 Dominguez[1469:10438] -canOpenURL: failed for URL: "jarvisdisney://" - error: "The operation couldn’t be completed. (OSStatus error -10814.)"
2023-01-10 18:53:37.639893+0100 Dominguez[1469:10438] [SessionAPI] DS Core Platform SDK v12.0.7 (54156)
2023-01-10 18:53:37.640515+0100 Dominguez[1469:10438] [SessionAPI] Configuration disney-svod-3d9324fc/production [debugging = false]
2023-01-10 18:53:37.647166+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.710635+0100 Dominguez[1469:10438] [SceneConfiguration] Info.plist contained no UIScene configuration dictionary (looking for configuration named "(no name)")
2023-01-10 18:53:37.781256+0100 Dominguez[1469:10438] [SceneConfiguration] Info.plist contained no UIScene configuration dictionary (looking for configuration named "(no name)")
2023-01-10 18:53:37.782014+0100 Dominguez[1469:10438] [ApplicationLifecycle] UIWindows were created prior to initial application activation. This may result in incorrect visual appearance.
2023-01-10 18:53:37.794650+0100 Dominguez[1469:10677] [ConfigurationService] get /bam-sdk/v4.0/disney-svod-3d9324fc/apple/v12.0.7/ios/ipad/prod.json: http 200, application/json, 71737 bytes
2023-01-10 18:53:37.824944+0100 Dominguez[1469:10438] [Socket] connecting to default
2023-01-10 18:53:37.851967+0100 Dominguez[1469:10438] [UIDevice.orientation] Device orientations are not supported in Mac Catalyst processes. Ignoring attempt to set UIDevice orientation to: 1
2023-01-10 18:53:37.908025+0100 Dominguez[1469:10438] -canOpenURL: failed for URL: "jarvisdisney://" - error: "The operation couldn’t be completed. (OSStatus error -10814.)"
2023-01-10 18:53:37.915264+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915384+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915404+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915424+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915444+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915479+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915502+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915517+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915532+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915548+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915568+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.915585+0100 Dominguez[1469:10438] [SessionAPI] Get session information
2023-01-10 18:53:37.934524+0100 Dominguez[1469:10438] [SessionAPI] Activate download session configuration
2023-01-10 18:53:37.938145+0100 Dominguez[1469:10438] [SessionAPI] Restore download session
2023-01-10 18:53:37.939813+0100 Dominguez[1469:10438] [MEL] Info 00:00:00 PCSAdaptor updateConfig ["ads"]
2023-01-10 18:53:37.965788+0100 Dominguez[1469:10438] [plugin] AddInstanceForFactory: No factory registered for id <CFUUID 0x60000021c4a0> F8BB1C28-BAE8-11D6-9C31-00039315CD46
2023-01-10 18:53:38.000669+0100 Dominguez[1469:10438] [SessionAPI] Reauthorize current session
2023-01-10 18:53:38.065580+0100 Dominguez[1469:10438] [Logging] [Error] <Chromecast.ChromecastDiscovery: 0x600000242be0> CastDiscovery.swift: getDevices() line: 111: [cast] ❌ Called getDevices with missing DiscoveryManager
2023-01-10 18:53:38.094775+0100 Dominguez[1469:10438] [General] The -[VSSubscriptionRegistrationCenter setCurrentSubscription:] method requires an entitlement.
2023-01-10 18:53:38.097371+0100 Dominguez[1469:10438] [General] (
	0   CoreFoundation                      0x00000001ac56b3f8 __exceptionPreprocess + 176
	1   libobjc.A.dylib                     0x00000001ac0b6ea8 objc_exception_throw + 60
	2   CoreFoundation                      0x00000001ac56b29c +[NSException exceptionWithName:reason:userInfo:] + 0
	3   VideoSubscriberAccount              0x00000001e21e303c -[VSSubscriptionRegistrationCenter setCurrentSubscription:] + 640
	4   DominguezCore                       0x000000010480286c DominguezCore + 5204076
	5   DominguezCore                       0x00000001048028a0 DominguezCore + 5204128
	6   RxSwift                             0x000000010320d73c RxSwift + 202556
	7   RxSwift                             0x000000010320dab8 RxSwift + 203448
	8   RxSwift                             0x0000000103224ea4 RxSwift + 298660
	9   RxSwift                             0x000000010320ef80 RxSwift + 208768
	10  RxSwift                             0x000000010320f138 RxSwift + 209208
	11  RxSwift                             0x0000000103224440 RxSwift + 296000
	12  RxSwift                             0x000000010325dd08 RxSwift + 531720
	13  RxSwift                             0x000000010325de18 RxSwift + 531992
	14  RxSwift                             0x0000000103224440 RxSwift + 296000
	15  RxSwift                             0x0000000103204f1c RxSwift + 167708
	16  RxSwift                             0x0000000103205570 RxSwift + 169328
	17  RxSwift                             0x000000010321e624 RxSwift + 271908
	18  RxSwift                             0x00000001032055e8 RxSwift + 169448
	19  RxSwift                             0x0000000103224440 RxSwift + 296000
	20  RxSwift                             0x00000001031e99c0 RxSwift + 55744
	21  RxSwift                             0x00000001031e9bc8 RxSwift + 56264
	22  RxSwift                             0x0000000103224440 RxSwift + 296000
	23  RxSwift                             0x000000010325dd08 RxSwift + 531720
	24  RxSwift                             0x000000010325de18 RxSwift + 531992
	25  RxSwift                             0x0000000103224440 RxSwift + 296000
	26  RxSwift                             0x0000000103219a8c RxSwift + 252556
	27  RxSwift                             0x0000000103219b6c RxSwift + 252780
	28  RxSwift                             0x00000001031fa520 RxSwift + 124192
	29  RxSwift                             0x00000001031f9ea0 RxSwift + 122528
	30  RxSwift                             0x000000010326bcbc RxSwift + 588988
	31  DominguezFoundation                 0x0000000100df565c DominguezFoundation + 251484
	32  RxSwift                             0x000000010320d73c RxSwift + 202556
	33  RxSwift                             0x000000010320dab8 RxSwift + 203448
	34  RxSwift                             0x0000000103224ea4 RxSwift + 298660
	35  RxSwift                             0x000000010320ef80 RxSwift + 208768
	36  RxSwift                             0x000000010320f138 RxSwift + 209208
	37  RxSwift                             0x0000000103249ea0 RxSwift + 450208
	38  RxSwift                             0x0000000103249bac RxSwift + 449452
	39  RxSwift                             0x000000010320d404 RxSwift + 201732
	40  DominguezFoundation                 0x0000000100df0454 DominguezFoundation + 230484
	41  RxSwift                             0x0000000103219c78 RxSwift + 253048
	42  RxSwift                             0x0000000103228838 RxSwift + 313400
	43  RxSwift                             0x000000010325df20 RxSwift + 532256
	44  RxSwift                             0x0000000103228838 RxSwift + 313400
	45  RxSwift                             0x00000001031e9cc4 RxSwift + 56516
	46  RxSwift                             0x0000000103228838 RxSwift + 313400
	47  RxSwift                             0x000000010324d724 RxSwift + 464676
	48  RxSwift                             0x000000010324dd90 RxSwift + 466320
	49  RxSwift                             0x0000000103228838 RxSwift + 313400
	50  RxSwift                             0x000000010325df20 RxSwift + 532256
	51  RxSwift                             0x0000000103228ca4 RxSwift + 314532
	52  RxSwift                             0x00000001032207b0 RxSwift + 280496
	53  RxSwift                             0x0000000103228910 RxSwift + 313616
	54  RxSwift                             0x000000010320d404 RxSwift + 201732
	55  DominguezCore                       0x0000000104802148 DominguezCore + 5202248
	56  DominguezCore                       0x000000010484e764 DominguezCore + 5515108
	57  Dominguez                           0x00000001000bda84 Dominguez + 776836
	58  Dominguez                           0x00000001000c0b28 Dominguez + 789288
	59  Dominguez                           0x00000001000eb1a0 Dominguez + 962976
	60  RxSwift                             0x000000010320d73c RxSwift + 202556
	61  RxSwift                             0x000000010320dab8 RxSwift + 203448
	62  RxSwift                             0x0000000103224ea4 RxSwift + 298660
	63  RxSwift                             0x000000010320ef80 RxSwift + 208768
	64  RxSwift                             0x000000010320f138 RxSwift + 209208
	65  RxSwift                             0x0000000103224440 RxSwift + 296000
	66  RxSwift                             0x00000001032229d4 RxSwift + 289236
	67  RxSwift                             0x0000000103222b14 RxSwift + 289556
	68  RxSwift                             0x0000000103226398 RxSwift + 304024
	69  RxSwift                             0x0000000103262e60 RxSwift + 552544
	70  RxSwift                             0x0000000103224fb0 RxSwift + 298928
	71  RxSwift                             0x0000000103226180 RxSwift + 303488
	72  RxCocoa                             0x00000001012cd124 RX_ensure_observing + 238684
	73  libswiftCore.dylib                  0x00000001ba275994 $sSTsE7forEachyyy7ElementQzKXEKF + 688
	74  RxCocoa                             0x00000001012cce1c RX_ensure_observing + 237908
	75  RxSwift                             0x000000010320ce54 RxSwift + 200276
	76  RxSwift                             0x0000000103224ea4 RxSwift + 298660
	77  RxSwift                             0x000000010320ef80 RxSwift + 208768
	78  RxSwift                             0x000000010320f138 RxSwift + 209208
	79  RxSwift                             0x0000000103224440 RxSwift + 296000
	80  RxSwift                             0x00000001032140e4 RxSwift + 229604
	81  RxSwift                             0x00000001032143c0 RxSwift + 230336
	82  RxSwift                             0x0000000103224440 RxSwift + 296000
	83  RxSwift                             0x0000000103222a04 RxSwift + 289284
	84  RxSwift                             0x0000000103222b14 RxSwift + 289556
	85  RxSwift                             0x0000000103226398 RxSwift + 304024
	86  RxSwift                             0x0000000103262e60 RxSwift + 552544
	87  RxSwift                             0x0000000103224fb0 RxSwift + 298928
	88  RxSwift                             0x0000000103226180 RxSwift + 303488
	89  RxSwift                             0x000000010326bcbc RxSwift + 588988
	90  RxRelay                             0x000000010025307c RxRelay + 28796
	91  Dominguez                           0x0000000100097530 Dominguez + 619824
	92  Dominguez                           0x000000010002fcf8 Dominguez + 195832
	93  UIKitCore                           0x00000001d6550664 -[UIViewController _setViewAppearState:isAnimating:] + 1108
	94  UIKitCore                           0x00000001d65f2878 -[UIViewController __viewDidAppear:] + 272
	95  UIKitCore                           0x00000001d65f2718 -[UIViewController _endAppearanceTransition:] + 204
	96  UIKitCore                           0x00000001d65f24fc __48-[UIPresentationController transitionDidFinish:]_block_invoke + 100
	97  UIKitCore                           0x00000001d7398fc0 -[_UIAfterCACommitBlock run] + 72
	98  UIKitCore                           0x00000001d739941c -[_UIAfterCACommitQueue flush] + 164
	99  UIKitCore                           0x00000001d64c9e48 _runAfterCACommitDeferredBlocks + 496
	100 UIKitCore                           0x00000001d64c9bc4 _cleanUpAfterCAFlushAndRunDeferredBlocks + 108
	101 UIKitCore                           0x00000001d64c9ae0 _afterCACommitHandler + 32
	102 CoreFoundation                      0x00000001ac4efdc8 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 36
	103 CoreFoundation                      0x00000001ac4efcb4 __CFRunLoopDoObservers + 532
	104 CoreFoundation                      0x00000001ac4ee8e4 CFRunLoopRunSpecific + 704
	105 HIToolbox                           0x00000001b5bc3fa0 RunCurrentEventLoopInMode + 292
	106 HIToolbox                           0x00000001b5bc3de4 ReceiveNextEventCommon + 672
	107 HIToolbox                           0x00000001b5bc3b2c _BlockUntilNextEventMatchingListInModeWithFilter + 72
	108 AppKit                              0x00000001af770424 _DPSNextEvent + 632
	109 AppKit                              0x00000001af76f5b4 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 728
	110 AppKit                              0x00000001af7639e4 -[NSApplication run] + 464
	111 AppKit                              0x00000001af73ae28 NSApplicationMain + 880
	112 AppKit                              0x00000001af994a44 +[NSWindow _savedFrameFromString:] + 0
	113 UIKitMacHelper                      0x00000001c2a22ba8 UINSApplicationMain + 988
	114 UIKitCore                           0x00000001d64b8934 UIApplicationMain + 148
	115 Dominguez                           0x0000000100007a38 Dominguez + 31288
	116 dyld                                0x00000001ac0e7e50 start + 2544
)
Process 1469 stopped
* thread #1, queue = 'com.apple.main-thread', stop reason = EXC_BREAKPOINT (code=1, subcode=0x1afa52ab4)
    frame #0: 0x00000001afa52ab4 AppKit`-[NSApplication _crashOnException:] + 240
AppKit`-[NSApplication _crashOnException:]:
->  0x1afa52ab4 <+240>: brk    #0x1
    0x1afa52ab8 <+244>: bl     0x1b0218be0               ; symbol stub for: objc_begin_catch
    0x1afa52abc <+248>: str    xzr, [sp]
    0x1afa52ac0 <+252>: mov    x0, #0x0
Target 0: (Dominguez) stopped.
Process 1469 launched: '/Users/moremac/Library/Containers/io.playcover.PlayCover/Disney+.app/Dominguez' (arm64)
(lldb) exit
```

**Comment by Amachik** [comment]
Also try to disable playtools injection

**Comment by LorenzoMorelli** [comment]
In this way the app doesn't crash but it shows a blue screen like it is loading without going forward.
Here a screenshot
<img width="1800" alt="Screenshot 2023-01-10 at 21 22 33" src="https://user-images.githubusercontent.com/24634833/211654373-723e5efb-94e6-4ca1-a6f3-eed02c300255.png">

**Comment by Amachik** [comment]
I will look into it once im home

**Comment by LorenzoMorelli** [comment]
Thank you for your support!

**Comment by IsaacMarovitz** [comment]
What version of Disney+ are you using? I cannot recreate this in PlayCover 2.0.2 running Disney+ 2.14.5 with SIP disabled + NVRAM args set

**Comment by Amachik** [comment]
U need to set nvargs for it to work

**Comment by LorenzoMorelli** [comment]
I'm running the version 2.14.5 of Disney+, I already disabled SIP.
Regarding NVRAM I found a solution using what described in this link [https://www.applegamingwiki.com/wiki/PlayCover#Install_Instructions](https://www.applegamingwiki.com/wiki/PlayCover#Install_Instructions).

It starts now! Thank you very much!

For the people who incurred in my same problems I also suggest to set the app resolution in the graphic setting on "App Default".
